libglusterfs/src/syscall.c

```diff
@@ -218,12 +218,12 @@ sys_unlink(const char *pathname)
 }
 
 int
-sys_unlinkat(int dfd, const char *pathname)
+sys_unlinkat(int dfd, const char *pathname, int flags)
 {
 #ifdef GF_SOLARIS_HOST_OS
     return FS_RET_CHECK0(solaris_unlinkat(dfd, pathname, 0), errno);
```

Comment:
That '0' shouldn't also be `flags` ? However all other references I've found in internet about this function only uses 2 arguments. I'm not sure if this compiles on solaris...

Reply:
I don't think compilation will fail, https://docs.oracle.com/cd/E36784_01/html/E36872/unlinkat-2.html, as per this man page it has supported 3 arguments.